.gitattributes

```diff
@@ -0,0 +1 @@
+pkg/agent/testdata/**/*  linguist-generated=true 
```

Comment:
This seems to make sure that anything inside pkg/agent/testdata is always auto-collapsed. But there is no github supported way of not having the files in a PR. 

There is a file filter option right under "commits" but since we have the "line33.sh" and most of our changes are also bash files, it wont hide everything. Its too bad that there is not a way to filter on a file path